gcloud-java-storage/src/main/java/com/google/gcloud/storage/StorageOptions.java

```diff
@@ -47,11 +83,27 @@ private Builder(StorageOptions options) {
       super(options);
     }
 
+    /**
+     * Sets the path delimiter for the storage service.
+     * @param pathDelimiter the path delimiter to set
+     * @return the builder.
+     */
     public Builder pathDelimiter(String pathDelimiter) {
       this.pathDelimiter = pathDelimiter;
       return this;
     }
 
+    /**
+     * Sets the time source for the storage service. The time source is used by `signUrl` to compute
```

Comment:
I am not sure we need to mention signUrl but rather say ... used for testing ... when an absolute time is calculated given a relative time. (and we can mention this option in the signURL method).